plugin.video.catchuptvandmore/resources/lib/channels/sk/joj.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```

Comment:
the empty __init__.py file above is not needed with Python3. Please remove.

Reply:
fixed